### PR TITLE
Updated README with a disclaimer and repo links

### PR DESCRIPTION
**Pull request by stepps00**
Updated README, with a few words about the change and new per-country repo links. Once this is merged, we can add a new README in each per-country repository that looks something like [this](https://gist.github.com/stepps00/5c02f5abbd8c4695ac43da6cf8d59881), along with issue templates.

Note that there are no additional branches or tags in this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,3 +321,266 @@ We've add a separate document called [README.KNOWN.KNOWNS.md](README.KNOWN.KNOWN
 
 * https://whosonfirst.org/
 * https://whosonfirst.org/download/
+
+## Repositories
+* [whosonfirst-data-admin-ad](https://github.com/whosonfirst-data/whosonfirst-data-admin-ad)
```
Comment (nvkelso) [inline README.md]: Nit: Please add line between repos header and first bullet point

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # whosonfirst-data
 
+## Disclaimer
+As of May 2019, the whosonfirst-data repository has split into per-country repositories. You can read more about that change [here](https://whosonfirst.org/blog/2019/05/09/changes/). While we encourage tracking all data-related issues in this repository, the data itself will live in the per-country repositories for the foreseeable future.
```
Comment (nvkelso) [inline README.md]: Nit: please add line

Comment (nvkelso) [reply README.md]: Let’s make it more explicate and say: “We still track all data... issues...”

**Comment by nvkelso** [comment]
While I know many country codes by heart, I still get confused by a few. Please add country name to these list items.

**Comment by stepps00** [comment]
I'll update README to address the comments and add a new commit that actually removes all data/files other than the README.

File: README.md (hunk 1 above)
Comment (stepps00) [reply README.md]: fixed in https://github.com/whosonfirst-data/whosonfirst-data/pull/1609/commits/bf2d6b32bb3451c8fd6a36119cfb632e2a5a8973

File: README.md (hunk 2 above)
Comment (stepps00) [reply README.md]: https://github.com/whosonfirst-data/whosonfirst-data/pull/1609/commits/bf2d6b32bb3451c8fd6a36119cfb632e2a5a8973

**Comment by stepps00** [comment]
https://github.com/whosonfirst-data/whosonfirst-data/pull/1609#issuecomment-492935534 taken care of in https://github.com/whosonfirst-data/whosonfirst-data/pull/1609/commits/808e6dcd4c64075cc40fea5186326a7db048a0fe.

**Comment by stepps00** [comment]
Changes to the README are in this branch, including updated formatting and country names next to each repository link.

This branch is ready for all other files to be purged.

**Pull request closed without merging**